jans-cedarling/cedarling/src/authz/mod.rs

```diff
@@ -389,46 +353,60 @@ struct ExecuteAuthorizeParameters<'a> {
 }
 
 /// Structure to hold entites created from tokens
-//
-// we can't use simple vector because we need use uid-s
-// from some entities to check authorizations
-#[derive(typed_builder::TypedBuilder)]
+#[derive(Clone)]
 pub struct AuthorizeEntitiesData {
-    pub workload_entity: Entity,
-    pub access_token: Entity,
-    pub id_token_entity: Entity,
-    pub userinfo_token: Entity,
-    pub user_entity: Entity,
-    pub resource_entity: Entity,
-    pub role_entities: Vec<Entity>,
+    pub workload: Entity,
```

Comment:
Why workload and user is not optional? It was discussed [here](https://chat.gluu.org/group/service_cedarling?msg=aNaTsfLMxZaMkRN6m)

Reply:
fix here [818bda4](https://github.com/JanssenProject/jans/pull/10436/commits/818bda4b3f5c92ca2e92c438d68abffe164be542)